static/src/stylesheets/module/_release-message.scss

```diff
@@ -523,10 +523,16 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
     }
 }
 
-.showPaypal {
+.show-paypal {
```

Comment:
according to BEM conventions a nicer way might be:

```sass
.site-message__message--show-paypal {
    .membership__paypal-logo {
        display: inline-block;
    }
   
    .membership__message-text {
        @include mq(desktop) {
            font-size: 25px;
        }
    }
}
```

you then don't need separate `.show-paypal` and `.show-paypal-message-text` classes, you can just apply a class to the parent and it's clearer what's going on